packages/grid/_modules_/grid/components/toolbar/GridToolbarExportContainer.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { unstable_useId as useId, useForkRef } from '@mui/material/utils';
+import MenuList from '@mui/material/MenuList';
+import { ButtonProps } from '@mui/material/Button';
+import { isHideMenuKey, isTabKey } from '../../utils/keyboardUtils';
+import { useGridApiContext } from '../../hooks/utils/useGridApiContext';
+import { GridMenu } from '../menu/GridMenu';
+import { useGridRootProps } from '../../hooks/utils/useGridRootProps';
+import { gridClasses } from '../../gridClasses';
+
+const GridToolbarExportContainer = React.forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
Should it be exported to let people build there custom formats ?

Reply:
Yes, I forget this `export`